src/App.js

```diff
@@ -1,67 +1,67 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react';
 import Navigation from './components/Navigation/Navigation';
 import Rank from './components/Rank/Rank';
 import ImageInputForm from './components/ImageInputForm/ImageInputForm.js';
 import FaceRecognition from './components/FaceRecognition/FaceRecognition.js';
 import Particles from 'react-particles-js';
 import Clarifai from 'clarifai';
 
-
 import './App.css';
 
-const app = new Clarifai.App({apiKey: '7008eee28f0040098f61cf606569b149'});
+const app = new Clarifai.App({ apiKey: '7008eee28f0040098f61cf606569b149' });
 
 // console.log(response.outputs[0].data.regions[0].region_info.bounding_box);
 
 class App extends Component {
-
-  constructor() {
-    super();
-    this.state = {
-      input: '',
-      imageUrl: '',
-      box: {},
-    }
-  }
+  state = {
```

Comment:
Using class properties, you usually don't need a constructor. This is pretty much the equivalent of the code, but less and less error prone, since if you had props for this component, you'd have to call super with `super(props)`

Reply:
cool, good to know!